pkgs/development/python-modules/gps3/default.nix

```
@@ -5,13 +5,13 @@
 
 buildPythonPackage rec {
   pname = "gps3";
-  version = "0.33.3";
+  version = "2017-11-01";
 
   src = fetchFromGitHub {
-    owner = "onkelbeh";
+    owner = "wadda";
     repo = pname;
-    rev = version;
-    sha256 = "0a0qpk7d2b1cld58qcdn6bxrkil6ascs51af01dy4p83062h1hi6";
+    rev = "91adcd7073b891b135b2a46d039ce2125cf09a09";
+    sha256 = "sha256-sVK61l8YunKAGFTSAq/m5aUGFfnizwhqTYbdznBIKfk=";
```

```suggestion
    hash = "sha256-sVK61l8YunKAGFTSAq/m5aUGFfnizwhqTYbdznBIKfk=";
```
see https://nixos.org/manual/nixpkgs/stable/#fetchurl

---

done